lib/models/gzipStats.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const fs = require('fs');
+const zlib = require('zlib');
+const util = require('util');
+const gzip = util.promisify(zlib.gzip);
+
+module.exports = function (file) {
+  let contentsBuffer = fs.readFileSync(file);
```

Comment:
shouldn't we use promisified version of `fs.readFile` to be consistent with code below

Reply:
It used to be sync version previously also here
https://github.com/ember-cli/ember-cli/blob/8b168056ccd18679a16aa29c3a2de1a20df0e368/lib/models/asset-size-printer.js#L66
I just moved it to a separate file.